### PR TITLE
feat: add input and image component

### PR DESCRIPTION
**Pull request by shaik-saad**
This pr includes the addition of:
- responsive image component
- round image component
- different type of inputs 

screen shots: 
cat-1
cat-2
input

### CODE REVIEW

File: components/input/input.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    margin-right: 1rem;
+}
+
+.form-field {
+    display: block;
+    width: 100%;
+    padding: 0.5rem 0.5rem;
+    border: 1px solid var(--sleek-gray-color-400);
+    border-radius: var(--border-radius-extra-small);
+    margin: 0.5rem 0
```
Comment (ruchiragoswami) [inline components/input/input.css]: closing semi-colon ; 
Line 13 :)

File: components/input/input.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    <link rel="stylesheet" href="../../css/variables.css">
+    <link rel="stylesheet" href="../utilities/typography/typography.css">
+    <link rel="stylesheet" href="../component.css">
+    <link rel="stylesheet" href="./input.css">
+</head>
+<body>
+
+    <div class="container">
+        <label class="form-label" for="">Email *</label>
+        <input class="form-field" type="email" name="" id="" required>
```
Comment (kart-c) [inline components/input/input.html]: don't use empty attributes

File: components/input/input.css [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+}
+
+.form-field:focus {
+    border: 1px solid var(--info-color);
+    outline: 0.25rem solid rgba(144, 205, 244, 0.3);
```
Comment (ruchiragoswami) [inline components/input/input.css:18]: My personal choice, for **reusability,** I even make variables for shadow color and outline color. Example,
--outline-color: rgba(144, 205, 244, 0.3) ;

File: components/input/input.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+        <label class="form-label" for="">Enter your query here: *</label>
+        <textarea class="form-field form-text-area" name="" id="" rows="5"></textarea>
```
Comment (kart-c) [inline components/input/input.html]: same, don't use empty attributes

File: components/input/input.css [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+.warning-field {
+    border: 1px solid var(--warning-color);
+}
+
+.success-text {
+    color: var(--success-color);
+}
+
+.success-field {
+    border: 1px solid var(--success-color)
```
Comment (ruchiragoswami) [inline components/input/input.css]: ;

**Review by kart-c — CHANGES_REQUESTED** [review]
Hello empty attributes should not be used, `id` and `name` are empty in lot of places

**Review by ruchiragoswami — APPROVED** [review]
1. Good work Saad. 
2. I made some comments in the files. 
3. Another overall comment is I personally like to use <section> tags wherever applicable (semantically) , and find a balance between div and section tags.

File: components/input/input.css (hunk 1 above)
Comment (shaik-saad) [reply components/input/input.css]: done

File: components/input/input.css (hunk 3 above)
Comment (shaik-saad) [reply components/input/input.css:18]: Yes Ruchi, I do it the same way. But as I'm using a separate branch for the variables, I'll have to switch to it, so I decided to hard code the values for now.

File: components/input/input.css (hunk 5 above)
Comment (shaik-saad) [reply components/input/input.css]: done

File: components/input/input.html (hunk 4 above)
Comment (shaik-saad) [reply components/input/input.html]: Done, with the changes.

File: components/input/input.html [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+        <input class="form-field warning-field" type="password" name="password-warning-example" id="password-warning-example">
+        <div class="form-text warning-text">Your password is weak</div>
+        <label class="form-label success-label" for="password-success-example">Password *</label>
+        <input class="form-field success-field" type="password" name="password-success-example" id="password-success-example">
+        <div class="form-text success-text">Your password is strong</div>
+    </div>
+
+    <div class="container input-container">
+        <label class="form-label" for="checkedRadioInput">
+            <input class="form-radio" type="radio" name="radio-example" id="checked-radio-input" checked>
```
Comment (kart-c) [inline components/input/input.html]: label `for` attribute and input `id` attribute is different

File: components/input/input.html [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+            Checked
+        </label>
+        <label class="form-label" for="uncheckedRadioInput">
+            <input class="form-radio" class="form-field" type="radio" name="radio-example" id="unchecked-radio-input">
```
Comment (kart-c) [inline components/input/input.html]: same, label `for` attribute and input `id` attribute is different

**Review by kart-c — CHANGES_REQUESTED** [review]
Please check all the `for` and `id` attributes, as this binds the label to the input

**Review by kart-c — APPROVED** [review]

**Pull request merged**